MaterialDesignThemes.Wpf/MaterialDesignThemes.Wpf.csproj

```diff
@@ -24,6 +24,9 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Xaml.Behaviors.Wpf" />
```

Comment:
This change will require an update to `MaterialDesignThemes.nuspec` to add this NuGet package to the list of dependencies for each of the target frameworks. 

Historically we have resisted taking any outside dependencies in the core package, however, I think this package should be an exception as it really is core to WPF.

Reply:
This is also what I mention in the PR description. I am not entirely sure of the syntax in that nuspec file, so help would be greatly appreciated.

Reply:
@Keboo I have now pushed some updates to the nuspec file. I opted to downgrade to an earlier version of `Microsoft.Xaml.Behaviors.Wpf` in an attempt to reach a broader audience; not sure if that is needed.

Also, this is probably something that should be excluded from `dependabot` automatic updates since I assume we would like this dependency to stay fixed?

Reply:
I think for now, I will still let dependabot update it. Given that the library does not update much, if they ever release a breaking change (likely with a 2.0 release) then that will be the point where we need to make code changes to support it, and the nuspec file will be updated then.

Reply:
@Keboo Do you know why the versions (lower- and upper-bound) in the nuspec do not match up with the ones shown on nuget.org (see screenshot in this comment thread)?

Reply:
Yea, in an effort to stop fighting with the NuGet versions, the [script that builds the NuGets](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/master/Scripts/BuildNugets.ps1) overrides the version in the NuSpec file